<center>
    <img src="https://s3-api.us-geo.objectstorage.softlayer.net/cf-courses-data/CognitiveClass/Logos/organization_logo/organization_logo.png" width="300" alt="cognitiveclass.ai logo"  />
</center>


#### Add your code below following the instructions given in the course


<h1>My Jupyter Notebook on IBM Watson Studio</h1>

__Name: Alexander James Lewis__
<br>
Desired Occupation: Statistician

_I want to learn more about data science because I am enjoy learning and applying mathematical skills, and I particularly like statistics and probability; which are major components of data science._

<h3>
    The following code calculates 22 divided by 7 to give a very rough approximation of the value of pi.
</h3>

In [1]:
22/7

3.142857142857143

Here is a list of other approximations of pi:
<ul>
    <li>
        From the <a href="https://en.wikipedia.org/wiki/Approximations_of_%CF%80">Shatapatha Brahmana</a>
        there is 339/108
    </li>
    <li>
        <a href="https://en.wikipedia.org/wiki/Approximations_of_%CF%80">Ptolemy</a>
        used 377/120
    </li>
</ul>
<table>
    <tr>
        <th>
            Comparing These Approximations of Pi
        </th>
    </tr>
    <tr>
        <td>
            Approximation
        </td>
        <td>
            Value
        </td>
    </tr>
    <tr>
        <td>
            22/7
        </td>
        <td>
            3.1428571429
        </td>
    </tr>
    <tr>
        <td>
            339/108
        </td>
        <td>
            3.1388888889
        </td>
    </tr>
    <tr>
        <td>
            377/120
        </td>
        <td>
            3.1416666667
        </td>
    </tr>
    <tr>
        <td>
            Current Approximation
        </td>
        <td>
            3.1415926536
        </td>
    </tr>
</table>